casts/free-podcasts-screencasts-en.md

```diff
@@ -3,6 +3,7 @@
 * [Android](#android)
 * [C#](#csharp)
 * [C++](#cpp)
+* [Clean Coding](#clean-code)
```

Comment:
use a more generic title.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
Okay thanks! Will try to fit it into another category. Or find more of the same to fit into clean code.